apps/server/src/modules/tool/context-external-tool/service/context-external-tool-validation.service.ts

```diff
@@ -46,7 +47,9 @@ export class ContextExternalToolValidationService {
 		);
 
 		if (duplicate.length > 0) {
-			throw new UnprocessableEntityException('Tool is already assigned.');
+			throw new ValidationError(
+				`tool_param_name_duplicate: A tool with the same name is already assigned to this course. Tool names must be unique within a course.`
```

Comment:
Please rename this.. i am missing a test change for the error? Is there no test then please create one